sw.js

```
@@ -0,0 +1,85 @@
+};
+
+self.addEventListener("install", (event) => {
+    event.waitUntil(addResourcesToCache(PRECACHE_URLS));
+});
+
+self.addEventListener("activate", (event) => {
+    const cacheWhitelist = [CACHE_NAME];
+    event.waitUntil(
+        caches.keys().then(cacheNames =>
+            Promise.all(
+                cacheNames.map(cacheName => {
```

Biome (2.4.6)

[error] This callback passed to map() iterable method should always return a value.

(lint/suspicious/useIterableCallbackReturn)